src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/card/card.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card shadow-sm"
+     [ngClass]="cardClass">
+  <div class="card-body align-items-center justify-content-center">
+    <h4 class="card-title m-4">
+      <a *ngIf="link; else noLinkTitle"
+         [routerLink]="link">{{ cardTitle }}</a>
+
+      <ng-template #noLinkTitle>
+        {{ cardTitle }}
+      </ng-template>
+    </h4>
+
+    <div class="card-text text-center"
+         [ngClass]="contentClass">
```

Comment:
Not a big issue, but for the sake of simplicity, and given that we are "projecting" the input contents raw into the card body, consumers of this component will rather style the body contents rather than providing the class here.
```html
<cd-card title="Foo">
  <div id="foo" class="card-foo">
    ...
  </div>
</cd-card>
```